src/rpcserver.cpp

```diff
@@ -193,11 +193,6 @@ string CRPCTable::help(string strCommand) const
             continue;
         if ((strCommand != "" || pcmd->category == "hidden") && strMethod != strCommand)
             continue;
-#ifdef ENABLE_WALLET
```

Comment:
No.
the `!pwalletMain` check (wallet disabled at runtime yes/no) will be done within the new function `EnsureWalletIsAvailable(bool)` :
https://github.com/bitcoin/bitcoin/pull/5992/files#diff-df7d84ff2f53fcb2a0dc15a3a51e55ceR40